README.md

```diff
@@ -171,10 +171,10 @@ print(settings.PIKACHU)
 # PIKACHU = 'Psyduck_is_not_fine'
 ```
 
-Метод ```temp_set_attributes``` не является потокобезопасным.
+Method ```temp_set_attributes``` is not thread-safe.
 
-## Список настроек
-Для получения списка настроек можно использовать методы `to_dict()`, `to_json`:
+## Settings list
```

Comment:
```suggestion
## Settings list

```
MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines